Misc

montjoie opened the pull request:
This PR fix two minor problem found on master:
- Some quoted ids were forgotten in README after hexid PR
- Clean unused code in lavalab-gen.py

And add healthchecks repository

montjoie committed:
Add healths check jobs

montjoie committed:
Clean argument parsing

lavalab-gen.py does not handle arguments, lets remove all arg parsing
stuff.

montjoie committed:
README: Fix vendor/product ID format

The format for vendor/product ID need to be prefixed by 0x.

khilman commented on lava-master/Dockerfile:
```diff
@@ -57,6 +57,8 @@ RUN a2enmod proxy \
 
 COPY configs/tftpd-hpa /etc/default/tftpd-hpa
 
+RUN git clone https://github.com/montjoie/lava-healthchecks.git
```
Please move these to BayLibre github instead of your personal one.

montjoie replied:
Done

montjoie committed:
Remove tty/stdin options

Keeping stdin open is not necessary since a long time.
Remove thoses unused options
